MvvmCross/Platforms/Ios/Core/MvxSceneDelegate.cs

```diff
@@ -0,0 +1,90 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MS-PL license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+using MvvmCross.Core;
+using MvvmCross.ViewModels;
+
+namespace MvvmCross.Platforms.Ios.Core;
+
+public abstract class MvxSceneDelegate : UIResponder, IUIWindowSceneDelegate, IMvxLifetime
+{
+    public event EventHandler<MvxLifetimeEventArgs>? LifetimeChanged;
+
+    [Export("window")] public UIWindow? Window { get; set; }
+
+    [Export("scene:willConnectToSession:options:")]
+    public virtual void WillConnect(
+        UIScene scene,
+        UISceneSession session,
+        UISceneConnectionOptions connectionOptions)
+    {
+        if (scene is UIWindowScene windowScene)
+        {
+            RegisterSetup();
+            Window = new UIWindow(windowScene);
+            MvxIosSetupSingleton
+                .EnsureSingletonAvailable(this, Window)
+                .EnsureInitialized();
+            RunAppStart();
+            FireLifetimeChanged(MvxLifetimeEvent.Launching);
+        }
+    }
+
+    [Export("sceneDidDisconnect:")]
+    public virtual void DidDisconnect(UIScene scene)
+    {
+    }
+
+    [Export("sceneDidBecomeActive:")]
+    public virtual void DidBecomeActive(UIScene scene)
+    {
+        FireLifetimeChanged(MvxLifetimeEvent.ActivatedFromMemory);
+    }
+
+    [Export("sceneWillResignActive:")]
+    public void WillResignActive(UIScene scene)
```

Comment:
Make virtual as well?

Reply:
Yeah, will have a look at that. Need to add some docs too on how to use, so will fix before release. Thanks for pointing this out :)